index.html

```diff
@@ -0,0 +1,14 @@
+<!Doctype html>
+<html>
+<head> <link rel="stylesheet" href="default.css">
+  <title>BankJoy</title>
+</head>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <body>
+    <div id= "contact">
```

Comment:
Try to use indentation to clearly show how the elements are nested:


```html
<body>
  <div id= "contact">
    <span><i class="fa fa-phone" aria-hidden="true"></i>(650) 429-8711</span>
    <span><i class="fa fa-envelope-open-o" aria-hidden="true"></i><a href="#">Email Us</a></span>
  </div>
</body>
```

instead of 

```html
<body>
    <div id= "contact">
    <span><i class="fa fa-phone" aria-hidden="true"></i>(650) 429-8711</span>
    <span><i class="fa fa-envelope-open-o" aria-hidden="true"></i>
      <a href="#">Email Us</a></span>
  </div>
  </body>
```